packages/push-notifications/samples/sample.ts

```diff
@@ -1,31 +1,27 @@
-import { setupAgent } from '../tests/utils/agent'
-
-import { PushNotificationsApnsModule } from '@aries-framework/push-notifications'
+import { setupAgentApns } from '../tests/utils/agent'
 
 /**
  * replace `a-valid-connection-id` with the connection id you want to interact with
  * See the tests for a more accurate implementation
  */
 const run = async () => {
   // Setup the agent
-  const agent = setupAgent({
-    name: 'aries push notifications agent',
-    publicDidSeed: '65748374657483920193747564738290',
+  const agent = await setupAgentApns({
+    name: 'aries apns push notifications agent',
   })
 
   // Inject the PushNotificationModule
   // NOTE: This has to be done before initializing the agent
-  const pushNotificationsApnsModule = agent.injectionContainer.resolve(PushNotificationsApnsModule)
+  const pushNotificationsApnsModule = agent.modules.pushNotificationsApns
```

Comment:
AH thats because if you go to the `setupAgentApns` in `packages/push-notifications/tests/utils/agent.ts` you'll find that there is no fcm module registered. If you register one on the agent you also get the fcm module

Reply:
AH thats because if you go to the setupAgentApns in packages/push-notifications/tests/utils/agent.ts you'll find that there is no fcm module registered. If you register one on the agent you also get the fcm module

Reply:
actually there is sth funky going on. i'll fix it

Reply:
nice catch - I forgot to register the Api on the Fcm module